tests/Integration/Behaviour/Features/Scenario/Security/security_configuration.feature

```diff
@@ -0,0 +1,39 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s security
+@reset-database-before-feature
+Feature: Security configuration form
+  PrestaShop allows BO users to manage Security configuration
+  As a BO user
+  I must be able to edit the security configuration
+  Clear the outdated sessions for customers and employees
+
+  Background:
+    Given there is a customer named "John Doe" whose email is "john.doe@prestashop.com"
+
+  Scenario: Play with general configuration
+    Given I specify following properties for security form
+      | token            | 0 |
+    And shop configuration for "PS_SECURITY_TOKEN" is set to 1
+    Then the token configuration should be enabled
+    When I submit the security form
```

Comment:
You mixed Behat domain tests with Symfony controller tests 😨 

I am afraid you're opening a door that will hurt us all.

Symfony controller tests are very fragile, even more when run from inside Behat.

Reply:
Can we mark these tests as "NOT DOMAIN" somewhere? Because all other Behat tests follow a different workflow so I think we need to keep the apart. I dont want people to start being confused between the two. Moreover behat is supposed to be about domain and domains dont care about forms and inputs and toggles. Domains are supposed to use ubiquitous language.

Reply:
I don't test the controller, I test the form handler only which is logical and recommended by gherkin users :thinking: 
I'm tired of this PR, if you want I can remove all tests related to this

Reply:
I was surprised by the handler used in behat as well but if it's not the controller itself it's acceptable (even if it seems a bit overkill to me, updating the Configuration directly would work fine)
The whole controller would be a no go though